packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -22,14 +23,37 @@ export interface NativeSelectProps
   /**
    * @internal
    */
-  testId?: string;
   optionLabel?: string;
+  testId?: string;
 }
-const StyledNativeSelect = styled.select<{
+const StyledNativeSelectWrapper = styled.div<{
+  disabled?: boolean;
+  hasError?: boolean;
+  isInverse?: boolean;
   theme: ThemeInterface;
+}>`
+  ${inputWrapperStyles}
+  padding: 0 ${props => props.theme.spaceScale.spacing03} 0 0;
+  svg {
+    color: ${props =>
+      props.isInverse && props.disabled
+        ? transparentize(0.6, props.theme.colors.neutral100)
+        : props.disabled
+        ? transparentize(0.4, props.theme.colors.neutral500)
+        : 'inherit'};
+    margin: 0 0 0 -${props => props.theme.spaceScale.spacing06};
+    pointer-events: none;
+    z-index: 1;
+  }
+`;
+
+const StyledNativeSelect = styled.select<{
   isInverse?: boolean;
+  theme: ThemeInterface;
 }>`
   ${inputBaseStyles};
+  height: 38px;
```

Comment:
Do we need to hard code the height? If so, where did this number come from? Usually our numbers are multiples of 4, so just wondering if this needs to be 40px instead. I'll defer to @orion-cengage

Reply:
Removed fixed height.